packages/binding.core/spec/eventBehaviors.ts

```diff
@@ -62,12 +62,37 @@ describe('Binding: Event', function () {
   })
 
   it('Should prevent default action', function () {
-    testNode.innerHTML = "<a href='http://www.example.com/' data-bind='event: { click: noop }'>hey</button>"
+    testNode.innerHTML = "<a href='http://www.example.com/' data-bind='event: { click: noop }'>hey</a>"
     applyBindings({ noop: function () {} }, testNode)
     triggerEvent(testNode.childNodes[0], 'click')
         // Assuming we haven't been redirected to http://www.example.com/, this spec has now passed
   })
 
+  it('Should allow default action by setting preventDefault:false', function () {
+    testNode.innerHTML = "<div data-bind='event: {click: test}'><a href='#' data-bind='event: {click: {preventDefault: false}}'>hey</a></div>"
```

Comment:
We could also test `preventDefault: true` here, though the code path for that is identical to the `unwrap()` tested below so I'm not concerned.